lib/Form/Type/ContentType/ContentTypeUpdateType.php

```diff
@@ -95,6 +99,61 @@ public function buildForm(FormBuilderInterface $builder, array $options)
             ->add('publishContentType', SubmitType::class, [
                 'label' => 'content_type.publish',
                 'disabled' => !$hasFieldDefinition,
-            ]);
+            ])
+            ->addEventListener(FormEvents::PRE_SET_DATA, [$this, 'onPreSetData'])
+            ->addEventListener(FormEvents::PRE_SUBMIT, [$this, 'onPreSubmit'])
+        ;
+    }
+
+    public function onPreSetData(FormEvent $event)
+    {
+        $form = $event->getForm();
+
+        /** @var ContentTypeData $data */
+        $data = $event->getData();
+        if ($data->mainLanguageCode === $data->usedLanguageCode) {
+            return;
+        }
+        $this->disableNotTranslatableFields($form);
```

Comment:
like in method below
```suggestion
        $this->disableNotTranslatableFields($event->getForm());
```